osu.Game.Rulesets.Osu/Mods/OsuModArrange.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) 2007-2018 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using System;
+using System.Collections.Generic;
+using osu.Framework.Extensions.IEnumerableExtensions;
+using osu.Framework.Graphics;
+using osu.Game.Graphics;
+using osu.Game.Rulesets.Mods;
+using osu.Game.Rulesets.Objects.Drawables;
+using osu.Game.Rulesets.Osu.Objects;
+using OpenTK;
+
+namespace osu.Game.Rulesets.Osu.Mods
+{
+    internal class OsuModArrange : Mod, IApplicableToDrawableHitObjects
+    {
+        public override string Name => "Arrange";
+        public override string ShortenedName => "Arrange";
+        public override FontAwesome Icon => FontAwesome.fa_arrows;
+        public override ModType Type => ModType.Fun;
+        public override string Description => "Everything rotates. EVERYTHING";
+        public override double ScoreMultiplier => 1.05;
```

Comment:
Since it's marked as unranked rn, it should just be `1`.

Reply:
We probably want a base `ModFun` class which seals `ScoreMultiplier` with a value of 1, seals `Ranked => false` etc.

Reply:
not sure if the ModFun this should be inside PR.
Will reset ScoreMultiplier to 1.